pkgs/development/libraries/science/math/osqp/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, cmake, lib }:
+
+stdenv.mkDerivation {
+  pname = "osqp";
+  version = "2020-07-17";
```

```suggestion
  version = "unstable-2020-07-17";
```
Commit & PR title should also be changed accordingly.

Why do you not want to use the latest release: https://github.com/oxfordcontrol/osqp/releases/tag/v0.6.0?

---

It seemed like there was a long time since the last release, so I suspected they might have fallen behind on releases.

---

I disagree. I would expect there's a reason for their branching/release schedule. If there's a hard reason why you want/need the new version, I'm happy to listen, but in general I prefer using the tagged release if possible.